pkg/diagnostics/diagnostics.go

```diff
@@ -67,6 +68,17 @@ func CmdAnalyze() *cobra.Command {
 	return cmd
 }
 
+// CmdReport returns a CLI command
+func CmdReport() *cobra.Command {
```

Comment:
I think it can be easier to add commands if the proxy status is a subcommand, I mean:
`noobaa diagnostics report <sub-command>`.
As I understand now `noobaa diagnostics report` is a single command to get the proxy details only.

Reply:
The report will host all the flow we want to report.
We should not cut it with subcommands.

Reply:
Updated the description. Thanks